privacyidea/api/radiusserver.py

```diff
@@ -127,9 +127,16 @@ def test():
     password = getParam(param, "password", required)
     dictionary = getParam(param, "dictionary",
                           default="/etc/privacyidea/dictionary")
-
-    r = test_radius(identifier, server, secret, user, password, port=port,
+    r = 0
+    for server in servers.split(','):
```

Comment:
Please do

    for server in [x.strip() for x in servers.split(','):

This way we will also be able to handle lists like

1.2.3.4:1812, 2.3.4.5:1645

which contain whitespaces.

Reply:
In the privacyidea//lib/tokens/radiustoken.py is a strip also